reactDashboard/src/components/Footer/Footer.css

```diff
@@ -66,3 +71,31 @@
 footer {
   padding-bottom: 30px;
 }
+
+.darkmode footer {
+  background-color: #121212;
+  padding-bottom: 30px;
+}
+
+.darkmode footer .sitemap .entry {
+  color: #f2f2f2;
+  font-weight: normal;
+}
+
+.darkmode footer .sitemap .title {
+  color: #c1def1;
+}
+
+.darkmode .footer-base {
+  background-color: #f2f2f2;
+  opacity: 0.75;
+  color: #121212;
+  margin: 0;
```

Comment:
Why are these positional/radius attributes darkmode specific?

Reply:
✅ 
@Boopster wanted the rounded corners in darkmode

Reply:
As discussed, that's worth commenting on in the css. Same for the non-colour changes elsewhere

Reply:
I've added a lightmode block above it so it makes more sense

Reply:
Still worth commenting on the darkmode rounded corners in the css

Reply:
added comments to stylesheet